frontend/src/lib/components/button/iconButton/IconButton.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import {Icon} from '$lib/components/icon';
+  import {ButtonBase} from '../base';
+  import type {IconButtonProps} from './IconButton.type';
+
+  type $$Props = IconButtonProps;
+
+  export let icon: $$Props['icon'];
+  export let title: $$Props['title'];
```

Comment:
Should we remove this title property, as the title will be passed in the `restProps`?

Reply:
See comment below.